api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -1089,19 +1171,27 @@ def _render_remove_siret_content(
                 "button_text": "Confirmer",
             }
         )
+    additional_data = {
+        "Structure": venue.managingOfferer.name,
+        "Offerer ID": venue.managingOfferer.id,
+    }
+    if feature.FeatureToggle.WIP_ENABLE_OFFER_ADDRESS.is_active():
+        additional_data["Partenaire technique"] = venue.name
```

Comment:
technique ou culturel ?
```suggestion
        additional_data["Partenaire culturel"] = venue.name
```